cx_clips_executive/src/cx_clips_executive/clips/skills.clp

```diff
@@ -76,8 +54,8 @@
 		  (bind ?action_params (str-cat ?action_params " " ?param-value)))
 	)
 	(bind ?id (sym-cat ?action-name "_" (gensym*)))
-	(printout logwarn "Calling mapped skill '" ?sks "'" crlf)
-	(printout logwarn "Calling skill '" ?action-name " " ?action_params"'" crlf)
+	(printout t "Calling mapped skill '" ?sks "'" crlf)
+	(printout t "of action '" ?action-name " " ?action_params"'" crlf)
```

Comment:
this printout is a bit wonky, why two lines?

Reply:
because it can be quite long